src/lookup_routing/readme lookup.md

```diff
@@ -0,0 +1,124 @@
+# Neural Network Substitution-based routing for the National Water Model version 3.0**

```

Comment:
I changed the file to use markdown headings. Please look through and make sure it looks good/accurate to what you meant when rendered.